server/src/internalClusterTest/java/org/elasticsearch/index/shard/RemoveCorruptedShardDataCommandIT.java

```diff
@@ -566,7 +566,8 @@ public void testResolvePath() throws Exception {
         for (String nodeName : nodeNames) {
             final Path indexPath = indexPathByNodeName.get(nodeName);
             final OptionSet options = parser.parse("--dir", indexPath.toAbsolutePath().toString());
-            command.findAndProcessShardPath(options, environmentByNodeName.get(nodeName), environmentByNodeName.get(nodeName).dataFiles(),
+            command.findAndProcessShardPath(options, environmentByNodeName.get(nodeName),
+                new Path[] { environmentByNodeName.get(nodeName).dataFile() },
```

Comment:
Is there potential for a follow up here to make this method parameter be single-valued?

Reply:
Yes. I'm working through essentially anything in the entire codebase that uses Path[].